packages/fast-data-utilities-react/src/__tests__/schemas/align-horizontal.schema.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Component with align horizontal",
+    "description": "A test component's schema definition.",
+    "type": "object",
+    "id": "alignHorizontal",
+    "properties": {
+        "alignHorizontal": {
+            "title": "Align horizontal",
```

Comment:
I get and like the way the name is align preceeds the type in code.  But, for title and description i would make it more readable.  "title": "Horizontal alignment"